sdk/Cargo.toml

```diff
@@ -74,22 +73,28 @@ url = "2.2.2"
 ureq = "2.4.0"
 instant = "0.1.0"
 openssl = { version = "0.10.31", features = ["vendored"], optional = true }
+sha2 = "0.9.5"
```

Comment:
Is there a reason why we aren't using the same version of `sha2` in WASM and non-WASM builds?

Reply:
I thought it broke a bunch of tests when I was developing but it seems good when I try it now - updating.